packages/odyssey-react/src/components/Box/_box-other.scss

```diff
@@ -33,3 +33,16 @@
 .userSelectAll {
   user-select: all;
 }
+
+/* Element Resets */
+
+.asFieldset {
```

Comment:
Correct! More context here: https://okta.slack.com/archives/G01JS0FE32P/p1642027324105000

Reply:
Two things: First, this class isn't being used anywhere in the PR yet which is confusing. Second, aren't many of these styles already present in Box as other utility classes already? I think we'll want to avoid building element specific classes like this and placing them all in `_box-other.scss`. As an alternative we can introspect which type of argument is provided to `as` if we get "fieldset" we can apply a collection of utility classes to get this same effect as the defaults and then allow any overrides a user provides to also take effect.

Reply:
> Two things: First, this class isn't being used anywhere in the PR yet which is confusing.

My bad, I removed it as I was debugging a TS issue, it has been re-added

Reply:
@arnoldsandoval-okta no worries thanks for clarifying!

Reply:
This has been removed. I will introduce a Fieldset component shortly.